README.md

```diff
@@ -40,6 +41,13 @@ then run `lando miip --group=<group name>` with the following:
 * migrate_nidirect_node_recipe
 * migrate_drupal_7_link
 
+then run the following individual migrations:
+lando miip node_contact
+lando miip nidirect_book
+
+finally, publish migrated content by running:
+lando drupal nidirect:migrate:post:publish_status
```

Comment:
Was this not implemented as an event listener?

Reply:
No, don't think so. (If it was, it isn't working, as I just ran a full set of migrations and most of the content was still not published)